### PR TITLE
Use external nextcloud packages and native scripts

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>

### CODE REVIEW

File: src/views/GroupSettings.vue [hunk 1]
```diff
@@ -75,8 +75,10 @@
 
 <script>
 import axios from 'nextcloud-axios'
+import InitialState from 'nextcloud-initial-state'
```
Comment (ChristophWurst) [inline src/views/GroupSettings.vue]:
```suggestion
import {loadState} from 'nextcloud-initial-state'
```
? :)

Comment (skjnldsv) [reply src/views/GroupSettings.vue]: Oups :p

**Comment by skjnldsv** [comment]
> `js/user_retention.js` is 14k lines. Is this a production build?

Did not realised js was shipped here! Fixed!

File: src/views/GroupSettings.vue [hunk 2]
```diff
@@ -75,8 +75,10 @@
 
 <script>
 import axios from 'nextcloud-axios'
+import { loadState } from 'nextcloud-initial-state'
+import linkToOCS from 'nextcloud-router'
```
Comment (ChristophWurst) [inline src/views/GroupSettings.vue]:
```suggestion
import {linkToOCS} from 'nextcloud-router'
```

have you actually tested this? :D

Comment (skjnldsv) [reply src/views/GroupSettings.vue]: Nope :D
Thanks for being here ;)

**Comment by skjnldsv** [comment]
Reduced the size by 45% ( 544KB => 300KB )

File: package.json [hunk 3]
```diff
@@ -16,6 +16,8 @@
     "crypto-js": "^3.1.9-1",
     "debounce": "^1.2.0",
     "nextcloud-axios": "^0.2.1",
+    "nextcloud-initial-state": "0.0.3",
+    "nextcloud-router": "0.0.9",
     "nextcloud-vue": "^0.12.3",
     "nextcloud-vue-collections": "^0.5.6",
```
Comment (skjnldsv) [inline package.json]: @nickvergessen what is this used for? :thinking:

File: src/views/GroupSettings.vue [hunk 4]
```diff
@@ -75,8 +75,10 @@
 
 <script>
 import axios from 'nextcloud-axios'
-import { Multiselect } from 'nextcloud-vue'
-import _ from 'lodash'
+import debounce from 'debounce'
+import Multiselect from 'nextcloud-vue/dist/Components/Multiselect'
+import { linkToOCS } from 'nextcloud-router'
```
Comment (ChristophWurst) [inline src/views/GroupSettings.vue]: I think you should really test this https://github.com/ChristophWurst/nextcloud-router/blob/166d5622d53c49290556d6e48b1338ae25085bb7/lib/index.ts#L37 :see_no_evil:

Comment (skjnldsv) [reply src/views/GroupSettings.vue]: Done! :D

**Pull request merged**